### DISCUSSION TITLE
pydeck line layer example question

### DISCUSSION THREAD

**Discussion by karosc** [visgl/deck.gl#5470] [Q&A]
Why does the [pydeck LineLayer Example](https://deckgl.readthedocs.io/en/latest/gallery/line_layer.html) have way fewer lines than the [deck.gl LineLayer example](https://deck.gl/examples/line-layer/)? From what I can see, they both use the same data.

**Comment by Pessimistress** [comment]
https://github.com/visgl/deck.gl/issues/4138
